NameLayerMain/src/vg/civcraft/mc/namelayer/command/commands/TransferGroup.java

```diff
@@ -52,12 +52,6 @@ public boolean execute(CommandSender sender, String[] args) {
 			return true;
 		}
 		
-		if (!gm.hasAccess(g, uuid, PermissionType.getPermission("TRANSFER")) && 
```

Comment:
I might be missing it but there is no check here for primary owner initiating transfer. Seems anyone in the group could initiate it?